study-builder/tenants/testboston/pages/login.html

```diff
@@ -56,10 +62,6 @@
         text-decoration: underline !important;
     }
 
-    #ddp-header {
-        padding-top: 30px;
-    }
-
```

Comment:
I added this padding because when you click "forgot password", the UI changes and there's an arrow icon in the top left corner. The padding was to make room for it.

<img width="521" alt="Screen Shot 2020-06-17 at 12 27 46 PM" src="https://user-images.githubusercontent.com/5713833/84924063-0beebe00-b096-11ea-8127-79701063cbb5.png">

Reply:
Oh, I see. Let me combine these two approaches.

Reply:
Okay, now it looks better on mobile and desktop.